vmdb/spec/models/ems_refresh/refreshers/kubernetes_refresher_spec.rb

```diff
@@ -29,8 +30,9 @@ def assert_table_counts
     ContainerNode.count.should         == 1
     Container.count.should             == 3
     ContainerService.count.should      == 6
-    ContainerPortConfig.count.should   == 3
+    ContainerPortConfig.count.should   == 2
```

Comment:
This change belongs to the new version of heapster we're using. It now uses only 2 ports instead of 3.